src/stories/Button.tsx

```diff
@@ -33,7 +33,7 @@ export const Button = ({
   backgroundColor,
   label,
   ...props
-}: ButtonProps) => {
+}: ButtonProps): ReactElement => {
```

Comment:
Instead of ReactElement, type the component instead with React.FC<>. Inside the brackets, add the prop types